### PR TITLE
Removes reference to elife-published bucket by itself

### PR DESCRIPTION
**Pull request by Jenniferstrej**
second step of:
1. change `ppp_cdn_bucket` to the new value. the bot will write twice to the same place
2. change the bot removing every reference to `published_bucket`, which is not necessary anymore
3. rename the setting `ppp_cdn_bucket` to something longer-lasting (we create a new value, leave both old and new key available for a while, eventually removing the old one)

### CODE REVIEW

**Comment by giorgiosironi** [comment]
The removal looks good, could you try to add this activities to `lint.sh` if they are not already there? Progressively trying to get better feedback in development about changes to them, but I'm not sure if they have other weird things inside to make them passing.

**Comment by Jenniferstrej** [comment]
@giorgiosironi - shall I add one by one? Or isn't better to do activity/activity_*.py? Is that because there are a lot of activities that we don't touch much and we would need to fix them first?

**Comment by giorgiosironi** [comment]
You are correct, there are lots of warnings like
```
E: 52,32: Instance of 'Session' has no 'get_value' member (no-member)
E: 56, 8: Instance of 'Session' has no 'store_value' member (no-member)
```
on many activities when running it on `activity/*.py`, so adding them one at a time is easier to perform fixes.

File: provider/storage_provider.py [hunk 1]
```diff
@@ -6,12 +6,12 @@
 import os
 
 
-def get_storage_context(*args):
+def StorageContext(*args):
```
Comment (giorgiosironi) [inline provider/storage_provider.py]: ah, nice workaround

**Pull request merged**